docs/introduction.md

```diff
@@ -0,0 +1,9 @@
+# Introduction
+
+This book describes how to use the Matrix IRC bridge, giving an overview of functions for users and server administrators. If you
+believe there are errors or omissions in the book, please open an issue over at [https://github.com/matrix-org/matrix-appservice-irc/issues](matrix-appservice-irc)
+or if you believe you have the know-how, create a PR.
```

Comment:
If we're not already dealing with spam or low-quality PRs, an encouragement to contribute is better FLOSS culture.

My "neutral" language suggestion:
```suggestion
or create a PR.
```

But it could also be an invitation to send in PRs.

Reply:
Works for me.